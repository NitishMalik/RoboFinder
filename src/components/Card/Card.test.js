import { shallow, mount, render } from "enzyme";
import Card from "./index";
import React from "react";

describe("Testing Card Component", () => {
  test("expect to render Card Compoent", () => {
    expect(shallow(<Card />).length).toBe(1);
    expect(shallow(<Card />).debug()).toMatchSnapshot();
  });
});
