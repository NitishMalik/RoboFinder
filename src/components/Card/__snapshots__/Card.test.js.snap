// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Card Component expect to render Card Compoent 1`] = `
"<div className=\\"tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5\\">
  <img alt=\\"robots\\" src=\\"https://robohash.org/undefined?size=200x200\\" />
  <div>
    <h2 />
    <h2 />
    <h2>
      Contact
    </h2>
  </div>
</div>"
`;
